rpe/engines/python.py

```diff
@@ -15,24 +15,32 @@
 
 import importlib.util
 import inspect
+import sys
 
 from rpe.policy import Evaluation, Policy
 
 
 class PythonPolicyEngine:
 
-    _policies={}
+    counter = 0
 
     def __init__(self, package_path):
+
+        self._policies={}
         self.package_path = package_path
+        PythonPolicyEngine.counter += 1
+        self.package_name = 'rpe.plugins.policies.engine_' + str(PythonPolicyEngine.counter)
+
         self._load_policies()
 
     def _load_policies(self):
         spec = importlib.util.spec_from_file_location(
-            "pypol",
+            self.package_name,
             "{}/__init__.py".format(self.package_path)
         )
         module = importlib.util.module_from_spec(spec)
+        sys.modules[self.package_name] = module
```

Comment:
does it make sense to update `PYTHONPATH` in the environment instead?

Reply:
I don't think so. If my path contains a `google.py` wont that conflict with the `google.` namespace? 

The goal there was to allow the python module/plugin to do relative imports, so I don't think thats what we want here. Adding it to the path wont allow relative imports. For example if I pass in `/etc/foo/policies` as my python policy path, and that contains and that looks like this

* /etc/foo/policies
* * __init__.py
* * my_policy.py

The `__init__.py` can import via `from .my_policy import [someclass]`. If we add `/etc/foo/policies` to the `PYTHONPATH` that wouldn't work, however the `__init__.py` COULD import via `from my_policy import [someclass]`

So back to my first comment in this post, if we also had a `google.py` in that directory, the `__init__.py` wouldn't be able to `from google import [someclass]`

I still get confused with imports, so if my logic above is wrong let me know. Thats my understanding though

Reply:
thanks for the help here - between this and the README update I think I'm good here.